airflow/settings.py

```diff
@@ -244,6 +245,7 @@ def configure_vars():
     MODEL_SERVERS = normalize_path(os.path.join(AIRFLOW_HOME, *[os.pardir, 'trained-models']))
     LIVY_CONF_PATH = deepcopy(HADOOP_CONFIGS_FOLDER)
     CHANGELOG_PATH = os.path.join((os.path.abspath(os.path.dirname(__file__))), 'changelog.yaml')    
+	LANGUAGE_SERVER_PATH = "/langserver/python"
```

Comment:
Read from the env variables, if not present, then use `"/langserver/python"` as default value.
Something like:
LANGUAGE_SERVER_PATH = os.environ.get('LANGUAGE_SERVER_PATH', "/langserver/python")

Reply:
AIso, incorrect indentation

Reply:
Okay